solvation_analysis/rdf_parser.py

```diff
@@ -1,18 +1,30 @@
 import numpy as np
-import numpy.polynomial.polynomial
-import matplotlib.pyplot as plt
 from scipy.interpolate import interp1d, UnivariateSpline
+import scipy
+import matplotlib.pyplot as plt
 
 
-def fit_data(bins, rdf):
-    f = UnivariateSpline(bins, rdf, k=4, s=0)
-    x = np.linspace(0, 10, num=100)
-    y = f(x)
-    return x, y
+def interpolate_rdf(bins, rdf, floor=0.05, cutoff=5):
```

Comment:
Just a notation thing, the opposite of `floor` is often called `ceil`.

Reply:
So these aren't actually opposites, though admittedly the naming I used here is a bit confusing. Floor is used to detect the leading edge of the first peak, so its a y-axis point. Cutoff is the end of the fitting region, so its a x-axis point. Very open to alternative naming.

Reply:
That makes sense, I'm happy to leave them as they are.